src/mongo_client.ts

```diff
@@ -431,13 +431,14 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> implements
       | 'extendedMetadata'
     >;
 
+  private driverInfos: DriverInfo[] = [];
+
```

Comment:
Should we go ahead and mark `additionalDriverInfo` deprecated on the options object, too?

Reply:
We can't mark it as deprecated on the options directly because we're using `Pick` and `Omit`.  But the deprecation does carry from the `MongoOptions` type

<img width="677" height="281" alt="Screenshot 2025-09-16 at 3 44 54 PM" src="https://github.com/user-attachments/assets/fea04534-ce5d-4d55-a09e-5437214afcad" />